src/wl-clipboard.cpp

```diff
@@ -0,0 +1,48 @@
+#include "wl-clipboard.h"
+
+#include <QProcess>
+#include <QStandardPaths>
+#include <QTextCodec>
+
+#include "logger.hpp"
+
+const QString wl_copy_path = QStandardPaths::findExecutable("wl-copy");
+const QString wl_paste_path = QStandardPaths::findExecutable("wl-paste");
+
+bool wl_copy_to_clipboard(const QString& text) {
```

Comment:
The prefix `wl_` is exclusively used by wayland API. This might be very misleading and also might be problematic when, in a future update, wayland introduces own version of `wl_copy_to_clipboard`. 

My suggestion is to use namespace:

```c++
namespace wayland_clipboard_tools {
bool copy_to_clipboard(const QString& text);
bool paste_clipboard(QString& outText);
}
```

To be consistent with file naming convention of this project, use names: `wayland_clipboard_tools.hpp` and `wayland_clipboard_tools.cpp`.